src/main.rs

```diff
@@ -1,30 +1,60 @@
-use ggez::{Context, ContextBuilder, GameResult};
+// Import necessary modules and traits from the ggez crate
 use ggez::event::{self, EventHandler};
-use ggez::input::keyboard::KeyCode;
 use ggez::graphics::{self, Color, Drawable, Text};
+use ggez::input::keyboard::KeyCode;
 use ggez::timer;
+use ggez::{Context, ContextBuilder, GameResult};
+
+// Import the Rng trait from the rand crate for random number generation
 use rand::Rng;
 
+/// Screen width in pixels
```

Comment:
Can you please include that floats are being used here.

Reply:
dude you are right